commands/context.js

```diff
@@ -508,7 +507,7 @@ async function showInfo(executedBy, strings, message, args) {
                     { name: "Edit", value: strings.info.edit },
                     { name: "View", value: strings.info.view },
```

Comment:
Please replace `view` with `link` here, since you've changed the string name.

Reply:
oh yikes, lemme fix